Add a fragment about translating code examples (#1382)
python/devguide: documentation/translating.rst

files in the root of the repository using the ``gettext_compact=0``
style.

How should I translate code examples?
-------------------------------------

> Translate values (i.e. string literals) and comments.
> Don't translate keywords and names, e.g. of variables, functions, classes, arguments and attributes.

Reviewer: ```suggestion
Translate values in code examples (i.e. string literals) and comments.
Don't translate keywords or names,
including variable, function, class, argument, and attribute names.
```
